.squot

```diff
@@ -10,4 +10,4 @@ OrderedDictionary {
 	'src\/FS-Tests-Disk.package' : #SquotCypressCodeSerializer,
 	'src\/FS-Tests-AnsiStreams.package' : #SquotCypressCodeSerializer,
 	'src\/BaselineOfFileSystem.package' : #SquotCypressCodeSerializer
-}
+}
```

Comment:
Don't ask me what Squot did here. After nothing else than forking the repo, cloning it via HTTPS and patching a single method, Squot also listed the following change here, which I excluded:
![image](https://user-images.githubusercontent.com/38782922/72160781-389ffa80-33bf-11ea-89dc-ff0ace4b5e6c.png)

Reply:
Wrote the table again without a terminating line feed. No problem.

objectsReplacedByNames is an implementation detail for the metadata associated with a tracked object. I didn't get around to think about hiding it, that's why it became visible when I added nodes for those metadata changes. `true` is the intended setting when the objects are stored outside of the image, so including it would have been in order.